lib/wallet.js

```diff
@@ -197,6 +197,9 @@ class WalletClient extends Client {
    */
 
   createWallet(id, options) {
+    if (id === null) {
```

Comment:
Here u need to use `==` to catch `undefined` as well. Also there's no need for braces (look at hsd codebase)

Reply:
Sorry Alex I think I led you astray on this one